### ISSUE TITLE
hover tool broken due to undefined roundingFunction

### ISSUE THREAD

**Issue by bryevdv** [bokeh/bokeh#6018]
The `sprints.py` returns this error in the JS console when any point is hovered:
```
TypeError: roundingFunction is not a function. (In 'roundingFunction(value * power)', 'roundingFunction' is undefined)
```

This is happening in this block of code in Numbro:
```
function format(input, formatString, language, roundingFunction) {
  if (language != null && language !== numbro.culture()) {
    numbro.setCulture(language);
  }
  return formatNumbro(
    Number(input),
    formatString != null     ? formatString : defaultFormat,
    roundingFunction != null ? Math.round   : roundingFunction);
}
```
we are not passing a `roundingFunction`, so it is `undefined`, and the last line does not catch that. 

@mattpap is this a change in our fork?

**Comment by bryevdv** [comment]
Yup, the last change to numbro looks responsible for this:

<img width="718" alt="screen shot 2017-03-19 at 16 49 04" src="https://cloud.githubusercontent.com/assets/1078448/24085128/ff137b4a-0cc3-11e7-9420-19c644cb2acc.png">

previously `undefined` would have resulted in `Math.round` being selected. 

@mattpap please prioritize making the change in Numbro and updating the BokehJS build to pull in the new commit hash. I'm not so familiar with that or I would do it right now.

**Comment by mattpap** [comment]
Numbro is already fixed and appropriate change do bokehjs is available in PR #5774.
